objects/obj_ground_teams/Alarm_0.gml

```diff
@@ -0,0 +1,31 @@
+if (feature.f_type == P_features.Ancient_Ruins){
+	newline = "Your marines warily stalk through into the entrance of the ruins";
+	newline_color = "red";
+	scr_newtext();
+	if (feature.ruins_race ==0){
+		feature.determine_race();
+           newline="Your marines descended into the ancient ruins, mapping them out as they go.  They quickly determine the ruins were once ";
+            if (feature.ruins_race=1) then newline+="a Space Marine fortification from earlier times.";
+            if (feature.ruins_race=2) then newline+="golden-age Imperial ruins, lost to time.";
+            if (feature.ruins_race=5) then newline+="a magnificent temple of the Imperial Cult.";
+            if (feature.ruins_race=6) then newline+="Eldar colonization structures from an unknown time.";
+            if (feature.ruins_race=10) then newline+="golden-age Imperial ruins, since decorated with spikes and bones."; 
+			scr_newtext();
+	}else{			
+		newline = "The ruins seem much unchange from the last exploration records"
```

Comment:
"The ruins seem ~~much~~ unchange**d** since the last **expedition**, **according to our records**."

or something similar